web/source/kmwnative.ts

```diff
@@ -80,86 +80,49 @@ if(!window['keyman']['initialized']) {
      * KeymanWeb 2 revised keyboard location specification:
      *  (a) absolute URL (includes ':') - load from specified URL
      *  (b) relative URL (starts with /, ./, ../) - load with respect to current page
-     *  (c) filename only (anything else) - prepend keyboards option to URL 
+     *  (c) filename only (anything else) - prepend keyboards option to URL
      *      (e.g. default keyboards option will be set by Cloud)
-     *           
-     * @param {string}  Lfilename  keyboard file name with optional prefix                     
-     */   
-    keymanweb.getKeyboardPath=function(Lfilename) {           
-      var rx=RegExp('^(([\\.]/)|([\\.][\\.]/)|(/))|(:)');   
+     *
+     * @param {string}  Lfilename  keyboard file name with optional prefix
+     */
+    keymanweb.getKeyboardPath=function(Lfilename) {
+      var rx=RegExp('^(([\\.]/)|([\\.][\\.]/)|(/))|(:)');
       return (rx.test(Lfilename) ? '' : keymanweb.options['keyboards']) + Lfilename;
     }
 
     /**
      * Align input fields (should not be needed with KMEI, KMEA), making them visible if previously hidden.
-     * 
+     *
      *  @param  {object}   eleList    A list of specific elements to align.  If nil, selects all elements.
-     * 
+     *
      **/
     keymanweb.alignInputs = function(eleList: HTMLElement[]) {
-      if(device.touchable) {
-        var domManager = keymanweb.domManager;
-        var processList: HTMLElement[] = [];
-
-        if(eleList) {
-          // Did the user specify the actual element or the touch-alias?
-          eleList.forEach(function(element: HTMLElement){
-            if(element.base) {
-              // It's a touch-alias element, which is what we wish to perform alignment on.
-              processList.push(element);
-            } else {
-              // This retrieves an element's touch-alias, should it exist.
-              let touchAlias = element['kmw_ip'] as HTMLDivElement;
-              if(touchAlias) {
-                processList.push(element['kmw_ip']);
-              }
-            }
-          });
-        } else {
-          processList = domManager.inputList;
-        }
-
-        processList.forEach(function(element: HTMLElement) {
-          if(dom.Utils.instanceof(element, "TouchAliasElement")) {
-            (element as com.keyman.dom.TouchAliasElement).updateInput();
-          }
-          element.style.visibility = 'visible';
-          if(element.base.textContent.length > 0) {
-            element.base.style.visibility = 'hidden';
-          }
-        })      
-      }
+      // no-op
```

Comment:
```suggestion
      // no-op: deprecated in 16.0, removing in 17.0
```

Reply:
This function is removed in #7360.  I could put the additional commentation in kmwbase.ts's stub-version, though.